OptimizelySDK/Optimizely.cs

```diff
@@ -338,54 +339,60 @@ private bool ValidateInputs(string datafile, bool skipJsonValidation)
         /// <summary>
         /// Sends conversion event to Optimizely.
         /// </summary>
-        /// <param name="eventKey">Event key representing the event which needs to be recorded</param>
+        /// <param name="eventKey">Event key representing the event (must not be null, empty, or whitespace)</param>
         /// <param name="userId">ID for user</param>
         /// <param name="userAttributes">Attributes of the user</param>
         /// <param name="eventTags">eventTags array Hash representing metadata associated with the event.</param>
         public void Track(string eventKey, string userId, UserAttributes userAttributes = null,
             EventTags eventTags = null
         )
         {
-            var config = ProjectConfigManager?.GetConfig();
-
-            if (config == null)
```

Comment:
can we remove this config ready check?

Reply:
If we're going to call `.GetEvent(eventKey)` later (L365 below on this older version) to check if the provided `eventKey` is in the datafile, I think we'd probably still need this check, right?